contracts/allowListExamples/AllowListOffChainManaged.sol

```diff
@@ -1,11 +1,11 @@
 pragma solidity >=0.6.8;
-import "@openzeppelin/contracts/access/Ownable.sol";
 import "../interfaces/AllowListVerifier.sol";
+import "../EasyAuction.sol";
```

Comment:
Do we have an interface for this contract? In case, we could use it instead of the full contract.

Reply:
Not there yet.